Update image docs and behaviour (#2774)
rotki/rotki: docs/usage_guide.rst

Currently rotki tracks your balance by querying the different supported protocols, exchanges and blockchains. If you manually add information about a trade your balances will not be updated since trades are not consulted when updating the accounts' balances. If you want to manually update your balance for an asset please refer to the :ref:`manual balances section <track_balances>`.

Adding ledger actions
=====================

With ledger actions you can add events that represent incomes, losses, expenses, etc. On the left sidebar click on History and then the Ledger Actions button from the dropdown menu.
You can provide a location, for example an exchange, a bank, a blockchain or others. For the action type you can select from:

* Income
* Loss
* Donation Received
* Expense
* Dividends Income
* Airdrop
* Gift
* Grant

.. image:: images/ledger_action.png
   :alt: Add a ledger action
   :align: center

For ledger actions you can optionally specify a rate and a asset for the rate. This is the rate linked to the asset for this action. If no rate is provided, the historical price at the date of the action is used.

> When generating a profit and loss report some ledger actions might be taxable in your country or not. To customize the list of taxable actions refer to the :ref:`ledger actions settings <ledger_action_settings>` section.

Reviewer: ```suggestion
When generating a profit and loss report some ledger actions might be taxable in your jurisdiction and some not. To customize the list of taxable actions refer to the :ref:`ledger actions settings <ledger_action_settings>` section.
```

Manually adding trades or other events
**************************************

> Adding historical events

Reviewer: No no. There is already a title for the section above.

Change 

> Manually adding trades or other events

to

Adding historical events.

And your "historical events"

to 

manual trades.

It should be obvious if you do a `make html` and go and read the generated docs locally.

========================

Rotki will pull all your trade history from the exchanges whenever it needs it. But most of us have probably also done some OTC trades or taxable events at some point. Such events could even just be mining tokens, depending on your jurisdiction, participating in an ICO or getting paid in crypto.

> On the left sidebar click on History and then the Trade button from the dropdown menu. This will take you to the Trades page. Clicking on the + symbol will open you a menu as the following one

Reviewer: ```suggestion
On the left sidebar click on History and then the Trade button from the dropdown menu. This will take you to the Trades page. Clicking on the ``+`` symbol will open a menu like the following.
```

.. image:: images/external_trade.png
   :alt: Add an external trade
   :align: center

> To add a new trade or taxable event, fill in all the fields and press the "Save" button.

Reviewer: ```suggestion
To add a new trade fill in all the fields and press the "Save" button.
```

> If there was a fee for the trade you should input it in the corresponding box and also enter the currency the fee was paid in. This field is optional so if the Fee was 0 you can leave this field empty. You can optionally provide additional notes or even links to blockchain explorers for each trade.

Reviewer: Hmm here you still need to explain a few things:

Explain that the `amount` is the amount of base asset bought/sold and that the `rate` is the rate of quote asset per base asset bought/sold. 

Before the fee explanation

> In the Trade page you can see a table of all your OTC trades. You can edit or delete a trade by clicking on the appropriate icon at the rightmost part of each trade under the "Actions" column.

Reviewer: ```suggestion
In the Trades page you can see a table of all your external trades. You can edit or delete a trade by clicking on the appropriate icon at the rightmost part of each trade under the "Actions" column.
```

> Currently rotki tracks your balance querying the different supported protocols, exchanges and blockchains. If you manually add information about a trade your balances won't be updated since trades are not consulted when updating the accounts' balances. If after adding a trade you want to update the accounts balances please refer to the :ref:`manual balances section <track_balances>`.

Reviewer: Nice note. This also definitely (along with the other common questions) needs to be part of the FAQ. Small rewording here:

```suggestion
Currently rotki tracks your balance by querying the different supported protocols, exchanges and blockchains. If you manually add information about a trade your balances will not be updated since trades are not consulted when updating the accounts' balances. If you want to manually update your balance for an asset please refer to the :ref:`manual balances section <track_balances>`.
```

Adding ledger actions
=====================

With ledger actions you can add events that represent incomes, losses, expenses, etc. On the left sidebar click on History and then the Ledger Actions button from the dropdown menu.
You can provide a location, for example an exchange, a bank, a blockchain or others. For the action type you can select from:

* Income
* Loss
* Donation Received
* Expense
* Dividends Income
* Airdrop
* Gift
* Grant

.. image:: images/ledger_action.png
   :alt: Add a ledger action
   :align: center

> For ledger actions you can specify a rate and a asset for the rate. This is the rate linked to the asset for this action. If no rate is provided, the historical price at the date of the action is used.

Reviewer: ```suggestion
For ledger actions you can optionally specify a rate and a asset for the rate. This is the rate linked to the asset for this action. If no rate is provided, the historical price at the date of the action is used.
```

> 

Reviewer: Also add a note and link to the settings section, explaining how these actions may be taken into account in the profit/loss report depending on the type of ledger action as described here: https://rotki.readthedocs.io/en/latest/usage_guide.html#ledger-action-settings
